05-structs.Rmd

```diff
@@ -81,13 +81,13 @@ task BwaMem {
   String read_group_id = "ID:" + base_file_name
   String sample_name = "SM:" + base_file_name
   String platform = "illumina"
-  String platform_info = "PL:" + platform   # Create the platform information
+  String platform_info = "PL:" + platform   ## Create the platform information
 
 
   command <<<
     set -eo pipefail
 
-    #can we iterate through a struct??
+    ## can we iterate through a struct??
```

Comment:
We'll want to remove this comment; it was an internal note for the authors when we were looking for a way to avoid having to manually declare every file we want to `mv` into the workdir.